apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -226,6 +229,13 @@ type VSHNPostgreSQLEncryption struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// VSHNPostgreSQLTLS contains TLS specific parameters
+type VSHNPostgreSQLTLS struct {
+	// Enabled specifies if the instance should use TLS for the instance.
```

Comment:
You could also add here that there will be no interruption in the service, if this value changes.